Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python as base
+
+# Perform common operations, dependency installation etc...
+RUN pip install poetry
+RUN pip install flask
+COPY pyproject.toml poetry.lock ./
+RUN poetry update
+RUN poetry install
+ENV WEBAPP_PORT=5000
+EXPOSE ${WEBAPP_PORT}
+COPY . .
+
+FROM base as production
+
+# Configure for production
+ENTRYPOINT ["poetry", "run", "flask", "run", "--host=0.0.0.0", "todo_app/app:create_app()"]
```

Comment:
This last argument looks like it's from the `gunicorn` command, but here it causes Flask confusion. If we want to try out Gunicorn then run `poetry add gunicorn` from your terminal (and check in/commit the changes to `pyproject.toml` and `poetry.lock`), and try changing this to:
```Dockerfile
ENTRYPOINT ["poetry", "run", "gunicorn", "--bind", "0.0.0.0", "todo_app.app:create_app()"]
```
Or if you prefer to stick with Flask, then just remove this final argument:
```Dockerfile
ENTRYPOINT ["poetry", "run", "flask", "run", "--host=0.0.0.0"]
```
Note:
* For dev we want to stick with Flask, so do update the entrypoint below to remove the last argument
* For prod, if you switch to Gunicorn then it will run on port 8000 by default, so we will need to switch your command to use `--publish 127.0.0.1:5000:8000`